Deployment.md

```diff
@@ -0,0 +1,40 @@
+# Deployments
+
+PerfCompare is hosted on Netlify, and is updated every time commits are pushed to the following branches:
+
+| Branch Name | URL                                               | Description                                                                                                   |
+| ----------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------- |
+| production  | https://perf.compare/                             | Production branch, updated every 1-2 weeks.                                                                   |
+| beta        | https://beta--mozilla-perfcompare.netlify.app/    | This is the current development branch.                                                                       |
+| staging     | https://staging--mozilla-perfcompare.netlify.app/ | Staging branch before releasing to beta (it's not in use currently)                                           |
+
+Every pull request will be deployed as well to a separate domain, whose link will be added automatically to the PR in a comment.
+
+## How to deploy beta to production
+
+The easiest by far is to
+[create a pull request on GitHub](https://github.com/mozilla/perfcompare/compare/production...beta?expand=1).
+It would be nice to write down the main changes in the PR description.
+
+After the PR is created all checks should run. When it's ready the PR can be
+merged. Be careful to always use the **create a merge commit** functionality,
+not *squash* or *rebase*, to keep a better history.
```

Comment:
The team prefers to squash the commits, however.

Reply:
That's totally fine when merging to the main branch, but not to production. 

If you squash all commits to 1 production commit:
- you can't easily know if a specific commit from the main branch is in production.
- you can't check it production has extra commits (which would be bad).
- I'm not sure how the next PR would look like: indeed the commits on production would be different than the commits on the source branch. I don't know if git/GitHub can deal with this. (Maybe?)

Hope this clears up this topic!